src/java.base/share/classes/java/io/Console.java

```diff
@@ -145,6 +146,124 @@ public Reader reader() {
         throw newUnsupportedOperationException();
     }
 
+    /**
+     * Writes a string representation of the specified object to this console's
+     * output stream, terminates the line and then flushes the console.
+     *
+     * <p> The effects of {@code con.println(obj)} are as if by calling
+     * {@snippet lang = java:
+     *     con.format("%s%n", obj)
+     * }
+     *
+     * @param  obj
+     *         An object whose string representation is to write.
```

Comment:
(editorial) No initial cap, no trailing period. We're still doing the sentence fragment thing, right? Also, end with "is to be written." Same for `print`.

Reply:
Yes, I was well aware of those outdated practices, but tried to follow them for the sake of consistency within the file.

I'll fix "is to be written."